docs/guides/hooks.md

```diff
@@ -0,0 +1,85 @@
+# Hooks
+Hooks exist so that users can latch on to certain Video.js lifecycle events
```

Comment:
maybe something like:
`Hooks exist so that users can hook into the video.js player lifecycle`